index.html

```diff
@@ -1,103 +1,137 @@
+</style>
 
-</section>
 
-<section id="experience">
-    <h2>Experience</h2>
+</head>
 
-       <ul>
-         <li>HumanPredictions</li> 
-         <li>Notre-Shop</li>
-         <li>Fishmen Fabrics</li>
-       </ul>
+<body> 
+  <div class="menu">
+    <header>
 
-</section>
+      <video autoplay loop muted width="720" height="480" class="home-video">
+         <source src="images/video.mp4" type="video/mp4">
+      </video>
 
 
-<section id="skills">
-   <h2>Skills</h2>
-   <ul class="skillSection">
-   </ul>
-     
-    <ul>
-        <li>HTML</li>
-        <li>CSS</li>
-        <li>JavaScript</li>
-     </ul>
+      <nav>
+        <a href="index.html">
+           <img  id="logo" src="images/logo.png"  alt="logo">
+        </a>
      
+        <ul>
+           <li><a href="#">Home</a></li> 
+           <li><a href="#about">About</a></li>
+           <li><a href="#experience">Experience</a></li>
+           <li><a href="#skills">Skills</a></li>
+        </ul>
+      </nav>
+    </header>
+  </div>
+  
+  <section id="about">
+     <h1>ABOUT</h1>
+     <img id="img-riri" src="images/riri.png" alt="riri">
+     <p id="about-me">Hi, My name is  Raíssa, I was born and raised in São Paulo Brazil. I moved to the USA in 2013, I now live in Chicago. I'm currently a student at Code the Dream, I am learning front-end development skills, and I hope to one day become a Software Developer.</p>  
+  </section>
+
+
+  <section id="experience">
+     <h1>EXPERIENCE</h1>
+     <table>
+      <tr>
+        <th>Role</th>
+        <th>Company</th>
+        <th>Date</th>
+      </tr>
+      
+      <tr>
+        <td>Data Resercher</td>
+        <td>Human Predictions</td>
+        <td>12/2020 - 09/2022</td>
+      </tr>
+      
+      <tr>
+        <td>Ecommerce Stylist</td>
+        <td>Notre Shop</td>
+        <td> 03/2018 - 03/2020</td>
+      </tr>
+    
+     <tr>
+        <td>Sales Associate</td>
+        <td>Fishmen Fabrics</td>
+        <td> 07/2017 - 03/2018</td>
+      </tr>
+    </table>
 
-</section>
-
-
-<section id="connect">
-    <h2>Connect</h2>
-
-      <ul>
-         <li>raissanfg@gmail.com</li>
-         <li>https://twitter.com/raissanfg</li>
-         <li>https://www.instagram.com/raissanfg</li>
-      </ul>
-
-</section>
-
-<section>
-   <h2>Leave a Message</h2>
-     <form name="leave_message"> 
-
-      <label class="field_label" for="name">Name</label><br>
-      <input type="text" name="name" required="true"><br>
-
-      <label class="field_label" for="email">Email</label><br>
-      <input type="email" name="email" required="true"><br>
-
-      <label class="field_label" for="message">Message</label><br>
-      <textarea name="message" required="true"></textarea> <br>
+  </section>
 
-     <button type="submit">Submit</button><br>
+  <section id="skills">
+    <h1>SKILLS</h1>
+    <ul>
+      <li class="skl">HTML</li>
+      <li class="skl">CSS</li>
+      <li class="skl">JAVASCRIPT</li>
+      <li class="skl">GIT</li> 
```

Comment:
Since you're adding these through javascript, you should not need to have them statically defined here in your HTML as well.